zerver/tests/test_invite.py

```diff
@@ -195,6 +196,7 @@ def invite(
             "/json/invites",
             {
                 "invitee_emails": invitee_emails,
+                "send_notification": "true" if send_notification else "false",
```

Comment:
I missed this before. The correct way to do this is `orjson.dumps(send_notification).decode()`.